config/environments/development.rb

```diff
@@ -44,18 +53,17 @@
   config.assets.quiet = true
 
   # Raises error for missing translations.
-  # config.action_view.raise_on_missing_translations = true
+  # config.i18n.raise_on_missing_translations = true
+
+  # Annotate rendered view with file names.
+  # config.action_view.annotate_rendered_view_with_filenames = true
 
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
-  # config.file_watcher = ActiveSupport::EventedFileUpdateChecker
-
-  config.active_job.queue_adapter = :inline
+  config.file_watcher = ActiveSupport::EventedFileUpdateChecker
 
-  # Tell Action Mailer not to deliver emails to the real world.
-  # The :file delivery method writes emails that would be sent to the
-  # application log file
-  config.action_mailer.delivery_method = :file
```

Comment:
I think we need to keep this configuration, as it will allow us to see the emails sent by the app when developing locally.

Reply:
Thanks @brucebolt I've added this back in.